src/main/java/org/stellar/sdk/xdr/ManageBuyOfferOp.java

```diff
@@ -53,19 +53,19 @@ public Price getPrice() {
   public void setPrice(Price value) {
     this.price = value;
   }
-  private Int64 offerID;
-  public Int64 getOfferID() {
+  private String64 offerID;
```

Comment:
In the latest [XDR definition file](https://github.com/stellar/stellar-core/blob/d99013e685/src/xdr/Stellar-transaction.x#L139L149), the offer id is still `int64`, so I don't think it should be changed to `String64` here.

Do you edit these XDR files by hand? Generally, we will not modify the XDR generated files directly, but use [xdrgen](https://github.com/stellar/xdrgen) to generate these codes.

Please let me know if I am not clear enough.